cl/lib/elasticsearch_utils.py

```diff
@@ -68,8 +70,23 @@ def build_fulltext_query(field: str, value: str) -> QueryString | None:
                     f"docketNumber:{docket_number}",
                     value,
                 )
-        return Q("query_string", query=value, fields=[field])
-    return None
+
+        q_should = [
+            Q(
+                "multi_match",
+                query=value,
+                fields=fields,
+                type="cross_fields",
+                operator="AND",
+                tie_breaker=0.3,
+            ),
+            Q("query_string", query=value, default_operator="AND"),
+        ]
+        for field in fields:
+            q_should.append(Q("match_phrase", **{field: {"query": value}}))
```

Comment:
Hey @albertisfu - what sort of tests did you do to determine that we need _both_ the `multi_match` _and_ the `match_phrase` queries? Did you try `multi_match` with `type="phrase"`?

I'm not saying it's wrong, but this results in a lot of repeated or nearly repeated queries and I _think_ we should be able to do this in multi_match by itself: https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-multi-match-query.html#type-best-fields

Reply:
Yeah, basically I checked that all the tests we added for the Solr version search also passed in the Elasticsearch version.  I just confirmed that using `type="phrase"` in the `multi_match` query also works properly, so I'll change it so the query is shorter, thank you!